client/components/Nav/NavDropdownMenu.jsx

```diff
@@ -22,8 +23,28 @@ export function useMenuProps(id) {
 function NavDropdownMenu({ id, title, children }) {
   const { isOpen, handlers } = useMenuProps(id);
 
+  let floatingProps;
+  if (id === 'file' || id === 'edit' || id === 'sketch' || id === 'help') {
```

Comment:
I would stay away from hard-coding which menus get which position.  If we want to support both directions then we can use a prop like `align="left"`.

That said, right now all menus are positioned from the bottom left. So I think we can use the same `placement: 'bottom-start'` for all.

The `floating-ui` package will handle shifting it over if it doesn't fit within the window.  I think we want the `shift` middleware for that?
https://floating-ui.com/docs/useFloating#visibility-optimizers
https://floating-ui.com/docs/shift

Reply:
using `shift({ mainAxis: true })` is working